Automattic/wp-calypso
# Generated Design Picker: Scrolling inside iframe

#### Changes proposed in this Pull Request

* This is another way to make elements stick to the viewport by setting the height of the iframe. It might not look good as the thumbnails are not responsive for now. The disadvantage I thought of is that there is an opportunity to have 2 scrollbars on the page. One is on the outer screen and the other is inside the iframe. Another disadvantage is that the height of the preview might be smaller as we try to keep one scrollbar inside the iframe. Otherwise, we can set the height of the preview to be `100vh - the height of the navbar with 48px padding-bottom - the gap of the page bottom`

Note that you can compare the result with https://github.com/Automattic/wp-calypso/pull/64110 which sticks the navbar, thumbnails, the browser header of the preview, and continue button.

https://user-images.githubusercontent.com/13596067/170671997-3923bd08-7004-4fd5-ab70-15fd8771f0ad.mov

#### Testing instructions



* Go to generated design picker: /setup/designSetup?siteSlug=<your_site>
* Scroll down to the navbar, thumbnails, browser header, and continue button are sticky
* Resize and scroll up and down again to check the UI won't be broken
* Switch to another design and scroll up and down to check the UI won't be broken



Related to https://github.com/Automattic/wp-calypso/issues/63794
